.github/workflows/bazel.yaml

```diff
@@ -127,41 +61,7 @@ jobs:
       ########################################
       - name: Update k8s deployments
         if: ${{ startsWith(github.ref, 'refs/tags/v') || startsWith(github.ref, 'refs/heads/container') || (github.ref == 'refs/heads/main') }}
-        env:
-          GITLAB_API_TOKEN: ${{ secrets.GITLAB_API_TOKEN }}
-        run: |
-            set -eExou pipefail
-
-            cd .git
-
-            # checkout branch
-            git clone "https://gitlab-ci-token:${GITLAB_API_TOKEN}@gitlab.com/dfinity-lab/private/k8s/k8s.git"
-
-            cd k8s
-            git config user.email "idx@dfinity.org"
-            git config user.name "IDX Automation"
-            git checkout -b "update-image-tag-${GITHUB_SHA}"
-
-            # Update the internal dashboard image refs
-            # this regex matches the first group (ie the image name) and uses \1
-            # called a back-reference to insert the first group matched, the second
-            # part is to match the 40 characters hash that we replace with the $GITHUB_SHA
-            sed -i "s~\(\([[:alpha:]]\|-\)\+\):[[:alnum:]]\{40\}~\1:${GITHUB_SHA}~g" bases/apps/mainnet-dashboard/statefulset-slack.yaml bases/apps/mainnet-dashboard/backend/base/deployment.yaml bases/apps/mainnet-dashboard/frontend/deployment.yaml
-
-            # commit changes if there are any
-            git add .
-            if git diff --cached --quiet; then
-                echo "No changes to commit."
-                exit 0
-            fi
-
-            # Push changes and create a new merge request
-            git commit -m "Updating container base image refs"
-            git push \
-                -o merge_request.create \
-                -o merge_request.title="[nomrbot] - Updating container image refs mainnet-dashboard [$GITHUB_SHA]" \
-                -o merge_request.description="Changes to the release repository - [here](https://github.com/dfinity/dre/commit/$GITHUB_SHA)" \
-                --force --set-upstream origin "update-image-tag-${GITHUB_SHA}"
+        uses: ./.github/workflows/update-deployments.yaml
```

Comment:
Is the plan to make this more generic than k8s deployments? If not, would be better to be more specific (include k8s in the file name).

Reply:
This is super specific, I think this cannot be made more generic for now. Will rename